apps/dropbox/package.json

```diff
@@ -16,7 +16,7 @@
     "start": "cross-env BROWSER=none react-scripts start",
     "build": "react-scripts build",
     "deploy": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${DEFINITIONS_ORG_ID} --definition-id 6YdAwxoPHopeTeuwh43UJu --token ${CONTENTFUL_CMA_TOKEN}",
-    "deploy:test": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${TEST_ORG_ID} --definition-id ${TEST_APP_ID} --token ${CONTENTFUL_CMA_TOKEN}"
+    "deploy:test": "contentful-app-scripts upload --ci --bundle-dir ./build --organization-id ${DEV_TESTING_ORG_ID} --definition-id 4OLkT2GLsM5HxBFGlnsR5q --token ${TEST_CMA_TOKEN}"
```

Comment:
Should these definitions be ENV variables?

Reply:
It's fine for the app id to be public - there's no security concerns